pkgs/development/python-modules/maestral/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv
+, fetchFromGitHub
+, python
+, sdnotify
+, systemd
+}:
+
+python.pkgs.buildPythonApplication rec {
```

Comment:
Done. I've also explicitly imported `pythonOlder` since that seems to follow that style as well.

Reply:
also meant to link this video on python packaging for nix https://www.youtube.com/watch?v=jXd-hkP4xnU&t=3s 

I think i mention briefly about the different between Package and Application

Reply:
It was a nice vid to watch, but it didn't cover that issue in particular. I'll keep these things in mind for the future though. The reason this package ended up in the state it was in is mostly because in the 0.6.4 version the maestral package was just a standalone application, so I ended up moving the nix code into python-modules wholesale. And it just worked so I didn't think to look at the other python modules to compare, which was my bad.

Please let me know if I can squash my new commit or if there are any other niggles.